frontend/dashboard/components/RepoList/ActionLinks.tsx

```diff
@@ -36,52 +36,56 @@ export const ActionLinks = ({ repo }: ActionLinksProps): React.ReactElement => {
 
   const repoFullName = repo.full_name;
   const [org, repoName] = repoFullName.split('/');
-  const isDataModelRepo = repoFullName.endsWith(DATA_MODEL_REPO_IDENTIFIER);
   const editUrl = getRepoEditUrl({ org, repo: repoName });
-  const editTextKey = t(isDataModelRepo ? 'dashboard.edit_data_models' : 'dashboard.edit_app', {
-    appName: repoName,
-  });
+
+  const giteaIconWithLink = (
+    <a href={repo.html_url}>
+      <span className={cn('fa fa-gitea', classes.giteaIcon)} />
+    </a>
+  );
+
+  const editIconWithLink = (
+    <a href={editUrl}>
+      <PencilIcon className={classes.nativeIcon} />
+    </a>
+  );
+
+  const dropdownAnchorButtonProps = {
```

Comment:
Maybe we could add a type to this object? 😄

Reply:
Fixed 😊